utils/filecache/cache.go

```diff
@@ -0,0 +1,200 @@
+package filecache
+
+import (
+	"context"
+	"errors"
+	"sync/atomic"
+	"time"
+
+	"github.com/ARM-software/golang-utils/utils/commonerrors"
+	fs "github.com/ARM-software/golang-utils/utils/filesystem"
```

Comment:
It is best to avoid renaming imports unless you have to. It isn't a rule but it is recommended by a lot of people.

There are a few cases where it is recommended but I think replacing a descriptive name with 2 letters is not one of them.

This google style guide is helpful https://google.github.io/styleguide/go/decisions.html#import-renaming